libs/operation_mode/src/operation_mode.cpp

```diff
@@ -0,0 +1,72 @@
+#include <ranges>
+
+#include <tfc/dbus/match_rules.hpp>
+#include <tfc/dbus/sdbusplus_meta.hpp>
+#include <tfc/operation_mode.hpp>
+#include <tfc/stx/concepts.hpp>
+#include <tfc/stx/string_view_join.hpp>
+
+#include <boost/asio.hpp>
+#include <sdbusplus/asio/connection.hpp>
+#include <sdbusplus/bus/match.hpp>
+#include <sdbusplus/message.hpp>
+
+namespace asio = boost::asio;
+
+namespace tfc::operation {
+
+[[maybe_unused]] static constexpr std::string_view mode_update_match_rule{
+  stx::string_view_join_v<tfc::dbus::match::rules::type::signal,
+                          tfc::dbus::match::rules::sender<tfc::operation::dbus::signal::update> >
+};
+
+static auto open_sd_bus() {
```

Comment:
This is also defined in a header above. Move to dbus util and utilize same function. Missing error check here.

Reply:
removed in next PR